modules/vaos/lib/tasks/vaos_tasks.rake

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Is this a placeholder?

Reply:
yes, part of the module generator, which is copied from the rails engine generator. It could be removed though.